### PR TITLE
HTTPCLIENT-2196 - Null check key value HttpCacheStorageEntry.

### PR DESCRIPTION
**Pull request by arturobernalg**

### CODE REVIEW

File: httpclient5-cache/src/main/java/org/apache/hc/client5/http/cache/HttpCacheStorageEntry.java [hunk 1]
```diff
@@ -45,7 +45,7 @@
     private final HttpCacheEntry content;
 
     public HttpCacheStorageEntry(final String key, final HttpCacheEntry content) {
-        this.key = key;
+        this.key = Args.notNull(key, "Cache Key");
```
Comment (michael-o) [inline httpclient5-cache/src/main/java/org/apache/hc/client5/http/cache/HttpCacheStorageEntry.java:48]: Use the same name as the method arg;  `key`

**Review by michael-o — CHANGES_REQUESTED** [review]

File: httpclient5-cache/src/main/java/org/apache/hc/client5/http/cache/HttpCacheStorageEntry.java (hunk 1 above)
Comment (arturobernalg) [reply httpclient5-cache/src/main/java/org/apache/hc/client5/http/cache/HttpCacheStorageEntry.java]: Changed. TY

**Comment by ok2c** [comment]
@arturobernalg I really do not want to be nasty, but why this particular place? There must be hundreds places where arg checks are missing. Why this place and not all others? Please, please finally start working on some feature and while doing so feel free to improve the code you touch, but these one line PRs you have been raising are just not very useful.

**Comment by arturobernalg** [comment]
> @arturobernalg I really do not want to be nasty, but why this particular place? There must be hundreds places where arg checks are missing. Why this place and not all others? Please, please finally start working on some feature and while doing so feel free to improve the code you touch, but these one line PRs you have been raising are just not very useful.

@ok2c  agree that there hundreds places where arg checks are missing, but don't you thing is time to face to problem and solved? I´m trying to improve the code. Every time that I found something susceptible to improvement I change it and I make a pr.I'm trying to fallow the guide that  you send in the mail, but keep in mind that is not an easy code and required a lot a functional expertise and a deep knowledge in the code and the standard. I'm working on it, but it's a slow process and need to go through in order to be more productive.
TY

**Comment by arturobernalg** [comment]
> > @arturobernalg I really do not want to be nasty, but why this particular place? There must be hundreds places where arg checks are missing. Why this place and not all others? Please, please finally start working on some feature and while doing so feel free to improve the code you touch, but these one line PRs you have been raising are just not very useful.
> 
> @ok2c agree that there hundreds places where arg checks are missing, but don't you thing is time to face to problem and solved? I´m trying to improve the code. Every time that I found something susceptible to improvement I change it and I make a pr.I'm trying to fallow the guide that you send in the mail, but keep in mind that is not an easy code and required a lot a functional expertise and a deep knowledge in the code and the standard. I'm working on it, but it's a slow process and need to go through in order to be more productive. TY

@ok2c and maybe for you it's an insignificant change, but improve the code and do it better for all the community. sn't that really the idea and essence of collaboration?

**Comment by ok2c** [comment]
@arturobernalg Please also close the JIRA.

**Comment by arturobernalg** [comment]
> @arturobernalg Please also close the JIRA.

Done

**Pull request closed without merging**